ethcore/wasm/src/runtime.rs

```diff
@@ -559,6 +547,39 @@ impl<'a, 'b> Runtime<'a, 'b> {
 		Ok(None)
 	}
 
+	fn sender(&mut self, context: InterpreterCallerContext)
+		-> Result<Option<interpreter::RuntimeValue>, InterpreterError>
+	{
+		let return_ptr = context.value_stack.pop_as::<i32>()? as u32;
```

Comment:
why doesn't `context.value_stack.pop_as::<u32>()?` work?

Reply:
it's not native webassembly type
there are only four `i32`, `i64`, `f32`, `f64`